cpp_exercise/src/lib/filehandlerlib.cc

```diff
@@ -76,9 +70,8 @@ std::vector<unsigned char> FileHandler::readFile(){
         smallbuffer.resize(ph);
```

Comment:
What is the objective of resizing it. If it's size is _buffer_size and you read all the size you can avoid this. Maybe you should use if (ph != _buffer_size && ph >= 0)

Reply:
I moved the resize to an inner if. if (ph != _buffer_size)